.gitignore

```diff
@@ -8,6 +8,7 @@ yarn-error.log*
 /test/e2e/reports/
 /src/styles/bulma-mods.css
 selenium-debug.log
+/test
```

Comment:
Please revert this, we want to be committing tests.

Reply:
sorry for that :o, I going to revert it.

Reply:
okay now ?